### PR TITLE
Adds Surface Buttons

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith @jbrundage @dtsnell4 

SVG AND LAYOUT SURFACE HTML BUTTONS

### CODE REVIEW

File: demos/dermatology.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
             overflow:hidden;
         }
     </style>
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```
Comment (mlzummo) [inline demos/dermatology.html]: remove

**Comment by mlzummo** [comment]
Need to make sure .remove is working correctly in both instances (especially with the extra button-container being added where it is in the layout/Surface)

**Comment by mlzummo** [comment]
padding margin and spacing need to be applied identically to svg/html

**Comment by GordonSmith** [comment]
Lint if failing.

**Pull request closed without merging**